include/NovelRT.h

```diff
@@ -3,13 +3,12 @@
 #ifndef NOVELRT_H
 #define NOVELRT_H
 
-//cstdlib
-#include <cmath>
-#include <cstdint>
-
 //stdlib
+#define __STDC_WANT_LIB_EXT1__
```

Comment:
What does this do?

Reply:
It exposes the "secure" variants of certain C stdlib functions (like `fopen_s`).

Reply:
secure in what sense?

Reply:
Basically the original C functions didn't take into account things like buffer overrun or certain types of errors.
Microsoft exposed "secure versions" of these functions (generally ending in `_s`) which started taking these into account.
These were later standardized as an official extension known as `__STDC_WANT_LIB_EXT1__` (Annex K - Bounds-checking interfaces in the current C language specification)
You can see https://docs.microsoft.com/en-us/cpp/c-runtime-library/security-features-in-the-crt?view=vs-2019 (and the C language spec) for more details.

Reply:
That's cool! Thanks for explaining that :D